source/grid/grid_generator.cc

```diff
@@ -235,7 +235,8 @@ namespace GridGenerator
         /**
          * Create a serial/parallel distributed triangulation.
          */
-        void create_triangulation(
+        void
+        create_triangulation(
```

Comment:
Is this what the indentation script does? Or did you make these changes by hand?

Reply:
I actually like this style better, and if the indentation script lets you do that, I'm in favor. But let's put this into a separate pull request.

Reply:
The indentation script did it. I do not indent by hand since then the CI usually complains.

Reply:
Apparently, something went wrong here. :-/ It should be fixed now.